docs/user_guide.md

```diff
@@ -252,6 +252,7 @@ analyzer arguments:
   -i SKIPFILE, --skip SKIPFILE
   --analyzers ANALYZER [ANALYZER ...]
   --add-compiler-defaults
+  --capture-analysis-output                      
```

Comment:
This should **NOT** be added here, as this argument is not part of the argument list of `check`.